index.html

```diff
@@ -197,30 +200,30 @@ <h3><a href="https://linkedin.com/in/ivan-martinez-von-halle/" class="connect-te
     <button class="resume-btn">Get my resume</button>
   </div>
 
-  <div class="about-languages">
-    <div class="languages-section">
+  <div class="about-knowlodges">
+    <div class="knowlodges-section">
       <h3 class="subt-about">Languages</h3>
       <img class="arrow" src="./images/languages-img/down-arrow.svg" alt="down-arrow" height="15" width="15">
     </div>
-    <ul class="languages-boxes">
-      <li class="language-box">
+    <ul class="knowlodges-boxes">
+      <li class="knowlodge-box">
         <img src="./images/languages-img/JS.svg" alt="javascript image" height="50" width="50">
         <p>JavaScript</p>
       </li>
-      <li class="language-box">
+      <li class="knowlodge-box">
         <img src="./images/languages-img/html.svg" alt="html image" height="50" width="50">
         <p>HTML</p>
       </li>
-      <li class="language-box">
+      <li class="knowlodge-box">
         <img src="./images/languages-img/css.svg" alt="css image" height="50" width="50">
         <p>CSS</p>
       </li>
     </ul>
-    <div class="languages-section frameworks">
+    <div class="knowlodges-section frameworks">
       <h3 class="subt-about">Frameworks</h3>
       <img class="arrow" src="./images/languages-img/right-arrow.svg" alt="right-arrow" height="15" width="15">
     </div>
-    <div class="languages-section skills">
+    <div class="knowlodges-section skills">
       <h3 class="subt-about">Skills</h3>
       <img class="arrow" src="./images/languages-img/right-arrow.svg" alt="right-arrow" height="15" width="15">
```

Comment:
- [ ] Your layout is indeed interesting, Yet as part of the requirement and the Figma design the left one should take 40% and the right one the remaining 60% of the page's width.

### your design 
![your](https://user-images.githubusercontent.com/69077061/193393120-7911f836-73e0-402c-a963-a22948b8332d.PNG)

### [Figma design ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848)
![figma](https://user-images.githubusercontent.com/69077061/193393127-b9437f1c-2189-4d86-bba9-2e0daa516e05.PNG)

Reply:
ok. Thanks